include/seqan3/core/add_enum_bitwise_operators.hpp

```diff
@@ -18,6 +18,78 @@
 
 namespace seqan3
 {
+/*!\interface seqan3::enum_bitwise_operators
+ * \brief You can expect these functions on all types that implement seqan3::add_enum_bitwise_operators.
+ */
+/*!\name Requirements for seqan3::enum_bitwise_operators
+ * \relates seqan3::enum_bitwise_operators
+ * \brief You can expect these member functions.
+ * \{
+ * \fn operator&(lhs_t lhs, rhs_t rhs)
+ * \brief Returns the binary `&` operator of lhs and rhs.
+ * \param lhs First enum.
+ * \param rhs Second enum.
+ *
```

Comment:
Do we need something like
```suggestion
 * \returns the conjunction of `lhs` and `rhs`.
```
? If yes, then please add it for each operator below as well.